src/HTMLElement/elements/HTMLTableRowElement.js

```diff
@@ -0,0 +1,177 @@
+import HTMLElement from '../../HTMLElement';
+
+export default class HTMLTableRowElement extends HTMLElement {
+    constructor() {
+        super();
+    }
+
+    /**
+     * Returns a live HTMLCollection containing the cells in the row. The HTMLCollection is live and is automatically
+     * updated when rows are added or removed.
+     * @todo Update array to HTMLCollection when it's implemented
+     * @member {Array.<Element>} HTMLTableRowElement#cells
+     */
+    get cells() {
+        return this._childNodeFind(child => child.tagName === 'td');
+    }
+
+    /**
+     * Is a DOMString containing one single character. This character is the one to align all the cell of a column on.
+     * It reflects the char and default to the decimal points associated with the language, e.g. '.' for English, or ','
+     * for French. This property was optional and was not very well supported.
+     * @member {string} HTMLTableRowElement#ch
+     */
+    get ch() {
```

Comment:
obsolete API, add `@deprecated` at least
I guess implementation could use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat, but doesn't worth it